.github/workflows/e2e-soak-trigger.yaml

```diff
@@ -0,0 +1,64 @@
+name: E2ESoakTrigger
+on:
+  schedule:
+    - cron: '0 */3 * * *'
```

Comment:
If this is every 3h, is this going to cause issues if any of the previous tests go over?

Reply:
All the tests have a 3h timeout. The thinking being here, tests should never have any overlap